src/sophys/common/plans/parallel_wrapper.py

```diff
@@ -1,8 +1,10 @@
+from bluesky import RunEngine as RE
+import asyncio
 from threading import Event, Thread
 
 
-def yield_msg(msg):
-    yield msg
+async def yield_msg(msg):
+    await asyncio.wait_for(RE._command_registry[msg[0]](msg), timeout=10**10)
```

Comment:
- is it safe to use this RE internal member?
- why use this timeout instead of `None`?

Reply:
And I don't think RE internal member is supposed to be used outside of the run engine class, but because running plans in parallel is not something that the bluesky framework supports natively and because I want to use the same functions of the run engine for the message, I chose to use it.